domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -73,13 +82,35 @@ class CameraXCameraUseCase @Inject constructor(
 
         previewUseCase.setSurfaceProvider(surfaceProvider)
 
-        cameraProvider.unbindAll()
-        camera = cameraProvider.bindToLifecycle(
-            lifecycleOwner,
-            cameraSelector,
-            previewUseCase,
-            imageCaptureUseCase
-        )
+        try {
+            cameraProvider.unbindAll()
+            camera = cameraProvider.bindToLifecycle(
+                lifecycleOwner,
+                cameraSelector,
+                previewUseCase,
+                imageCaptureUseCase
+            )
+        } catch (e: Exception) {
+            Log.e(TAG, "UseCase binding failed", e)
```

Comment:
Do you want to rethrow the exception or a more specific typed exception so that things can be handled correctly upstream (eg. retry or show an error)?

Reply:
This code isn't here anymore after Trevor's migration to ProcessCameraProvider